README.md

```diff
@@ -97,6 +98,17 @@ export VK_COMPILE_TIME_LOG=<log-file-path>
 
 To check if the layers are enabled, you can run a sample Vulkan application (such as `vkcube`) and look for the generated layer log files, or check the [Vulkan loader logs](https://github.com/KhronosGroup/Vulkan-Loader/blob/master/docs/LoaderInterfaceArchitecture.md#table-of-debug-environment-variables).
 
+### Log formats:
+
+To visualize the logs, we use multiple trace viewers such as the python scripts in the `scripts/` directory and Perfetto (https://ui.perfetto.dev/). To suit the input format requirements of each logger, we support logging in various formats such as CSV and Chrome Trace Event. see:
+https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview
+
+Our logging system also supports custom formats (e.g., `sample_output/events.log`).
```

Comment:
You can link to the file: https://github.blog/2013-01-31-relative-links-in-markup-files/